flatpak/build-flatpak-bundle.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/bash
```

Comment:
Would it be possible to provide an `sh` compatible version of this? Usually this just means avoiding any bash-specific commands and changing this to `/bin/sh`